### PR TITLE
Modifier

### PR DESCRIPTION
**Pull request by AlbyGiaco**
feat(modifier): adding modifier to position manager, onlyOwner, onlyGov, and make actions add remove replace only by gov

### CODE REVIEW

File: contracts/Registry.sol [hunk 1]
```diff
@@ -7,12 +7,15 @@ import '../interfaces/IRegistry.sol';
 /// @title Stores all the modules addresses
 contract Registry is IRegistry {
     address public governance;
+    address public override keeper;
```
Comment (AlbyGiaco) [inline contracts/Registry.sol:10]: @SBfin https://bridgedefi.atlassian.net/browse/ORB-349?atlOrigin=eyJpIjoiYmE2YzY5MDg0NzM2NDQzMWI0MmI2Yjk2NzNmY2FiZjAiLCJwIjoiaiJ9

File: contracts/PositionManagerFactory.sol [hunk 2]
```diff
@@ -22,12 +22,19 @@ contract PositionManagerFactory is IPositionManagerFactory {
         address _diamondCutFacet,
         address _uniswapAddressHolderAddress,
         address _registryAddress,
-        address _aaveAddressHolderAddress
+        address _aaveAddressHolderAddress,
+        address _governance
```
Comment (RiccardoGalbusera) [inline contracts/PositionManagerFactory.sol:26]: Isn't it better if we fetch governance somewhere? For example registry

File: contracts/utils/Storage.sol [hunk 3]
```diff
@@ -28,6 +28,7 @@ struct StorageStruct {
     IUniswapAddressHolder uniswapAddressHolder;
     address owner;
     IRegistry registry;
+    address governance;
```
Comment (RiccardoGalbusera) [inline contracts/utils/Storage.sol:31]: maybe put the two addresses togheter for clarity purposes?

File: contracts/utils/Storage.sol [hunk 4]
```diff
@@ -50,6 +51,11 @@ library PositionManagerStorage {
         emit OwnershipTransferred(previousOwner, _newOwner);
     }
 
+    function enforceIsContractOwner() internal view {
+        StorageStruct storage ds = getStorage();
+        require(msg.sender == ds.governance, 'LibDiamond: Must be contract owner');
```
Comment (RiccardoGalbusera) [inline contracts/utils/Storage.sol:56]: 'Contract::function: error message'

**Review by RiccardoGalbusera — APPROVED** [review]

**Review by nicolas-racchi — APPROVED** [review]

**Review by luduvigo — CHANGES_REQUESTED** [review]

**Review by luduvigo — APPROVED** [review]

**Pull request merged**